app/src/main/res/values/strings.xml

```diff
@@ -242,41 +242,43 @@
     <string name="login_enter_encryption_details">Secret Encryption Password</string>
     <string name="login_encryption_account_label">Account:</string>
     <string name="login_service_details_description">This is your login password, *not* your encryption password!</string>
-    <string name="login_forgot_password">Forgot password?</string>
+    <string name="login_forgot_password">Forgot your password?</string>
 
-    <string name="login_wrong_username_or_password">Couldn\'t authenticate (probably wrong username or password).\nHave you registered?</string>
+    <string name="login_configuration_detection">Setup detection</string>
+    <string name="login_querying_server">Please wait, querying server…</string>
+    <string name="login_wrong_username_or_password">Could not log in (probably wrong username or password).\nHave you registered?</string>
     <string name="login_view_logs">View logs</string>
 
     <string name="setting_up_encryption">Setting up encryption</string>
     <string name="setting_up_encryption_content">Please wait, setting up encryption…</string>
 
-    <string name="account_creation_failed">Account creation failed</string>
+    <string name="account_creation_failed">Could not create account</string>
     <string name="wrong_encryption_password">Wrong encryption password</string>
-    <string name="wrong_encryption_password_content">Got an integrity error while accessing your account, which most likely means you put in the wrong encryption password.\nPlease note that the username is case sensitive, so please also try different capitalizations, for example make the first character uppercase.\n\nError: %s</string>
+    <string name="wrong_encryption_password_content">Could not access your account. Most likely you entered the wrong encryption password.\nPlease note that the username is case sensitive, so please also try different capitalizations, for example make the first character uppercase.\n\nError: %s</string>
 
     <string name="signup_title">Enter Signup Details</string>
     <string name="signup_password_restrictions">Password should be at least 8 characters long</string>
 
     <!-- ChangeEncryptionPasswordActivity -->
     <string name="change_encryption_password_title">Change Encryption Password</string>
-    <string name="change_encryption_password_extra_info">Please don\'t use this tool if you believe your encryption password has been compromised. Contact support instead.</string>
+    <string name="change_encryption_password_extra_info">Please don\'t use this tool if you believe your encryption password is in the wrong hands. Contact support instead.</string>
```

Comment:
I prefer how it was before.

Reply:
Ask some people what "compromised" means in terms of passwords.
Ask some people what "in the wrong hands" means in terms of passwords. I imagine the latter not only gets more recognition and introduces less fear in users, but also avoids awkward translations of a word that isn't common.

Reply:
I think it's a common word with a well understood meaning. English is a rich language, and while I'm OK with avoiding the more esoteric parts of it, I think it's a bad idea to only use the 5000 most common words. Compromised is the correct word here. At least better than the suggested alternative.

Reply:
@tasn What regular user did you ask? Do you want those?
To the contrary it is a fantastic idea. Right now only the Turkish and Norwegian Bokmål translation even has completion. And, surprise surprise, the coverage in other languages doesn't extend to the "compromised" string
https://hosted.weblate.org/translate/etesync/android/en/?checksum=f5bd2cdf38910fea

If by linguistic style regular users aren't included, then by all means. It isn't an easy word to translate, nor is it common
(Other than EteSync) it is only found in Dino, pypa, F-Droid, OpenWrt, Debian Handbook, and phpMyAdmin. The latter 4 are not good examples of localization, and it is a selection of highly technical projects.
https://hosted.weblate.org/search/?page=1&limit=100&q=+source%3Acompromised+&sort_by=-priority%2Cposition

To decimate users further, try serving them "post processing" and "preparing for fetch" "case sensitive" "uppercase" "read-only" and see how many you are left with. Those are all in here. Arguably none of those are in https://www.oxfordlearnersdictionaries.com/wordlists/oxford3000-5000
It is going to be fewer users for no good reason at all.
How much do you like relying on only context when trying to do something you aren't good at?

Bonus, why not make "Would you like to revoke %s" "rescind %s" to drive the point home? If it is a certificate that is nullified, it isn't actually revoked. Yes, it is somehow still called revocation, fun for everyone.

Let's not get into the breadth of English, lest we forget how poor this all was a minute ago.

Reply:
The reason why compromised is not translated in most languages for EteSync is because we inherited a lot of translations from DAVx5, and this word is not one of them.

As I said, I don't want to limit to only the 5000 most common words. Sure, it's better when possible, but I wouldn't want to cripple the English version by using weird word constructs or incorrect terms just to make translation be more directly mapped. What you are suggesting is being incorrect so it's easier for translators to do their jobs. I don't think it's a good idea, and even if it is, it's not a path I'd like to go down.

Reply:
Why is it either incorrect or crippled? It is already in the translation.

The stringbase isn't limited to 5000 at all, it is a number you introduced, but it isn't inconsequential. What I am suggesting, (other than make it easier for translators to translate, helping it be easier for users to understand much of anything in their languages) is not _compromise_ the English variant when those aren't even available. Precisely crippling the English variant is what having "compromised" does. It increases the requirement of prior technical knowledge for no good reason.

There are no style-points to be had here. It is full of errors, and probably still is.
It is very possible to have good language in a limited set of words, however, expanding the arsenal of words has not only a natural drawback, but also is far, far less important than improving the language.

Try the "compromised" string on some users, and discover it decreases confidence even for native speakers vis-à-vis the alternative.

Reply:
This whole discussion is getting a bit tiring for me. It's obvious that we have different thoughts on this matter and reiterating our points won't get us nowhere. I think compromise is the correct term and I think "is in the wrong hands" is bad for a few reasons. Maybe you can find another word/term that is clear and addresses my concerns:
1. "In the wrong hands" feels clunky, imprecise and vague.
2. They should also not use this if it *could* have gotten to the wrong hands, so revealed anywhere, if no-one saw it. Think for example if I put my password here, and then edit it to be removed. I don't believe it got to the wrong hands, but it was definitely compromised.

That's the problem with using imprecise terminology. It looks good enough on first glance but it's not actually correct so fails scrutiny.

As for your authentication example: I was OK with it because "authentication" was generic, and "session expired" was more precise, unlike this case.

Reply:
Not "to the wrong hands" "into the wrong hands". Ah, I messed that up too.
Fair point about possibility. I argue it is then a compromised as a password, but that it could or could not be actually compromised. The distinction is not something I would bank on. Changing the string around.

Edit: Actually https://idioms.thefreedictionary.com/fall+into+the+wrong+hands is the correct idiom, but it is maybe a bit clunky.

Reply:
Better, but clunky. Maybe we should try something in the lines of "revealed" or "exposed" or something. I have no idea really, I'm just trying to think of synonyms for compromised...

Reply:
Better, but know still feels weird. "Could have gained access" maybe?

Reply:
I thought about it, went with "someone else could know your encryption password" I like "could have gained access" too.

Reply:
Looks fine now. One more thought though: since there's now only one password, what do you think, should we drop the word "encryption" from there? The title should still include it (since it should only be used in v1), but the content is used in v2 too. What do you think?

I can't wait to get rid of v1 strings. :)